src/http_ffi.erl

```diff
@@ -53,6 +53,7 @@ serve(Port) ->
             {port, Port},
             {default_type, "text/html"},
             {mime_types, mime_types()},
+            {customize, ?MODULE},
```

Comment:
Woah what is this?

Reply:
Ah I see, we're referring to the module itself because it implements `httpd_custom_api`: https://www.erlang.org/doc/man/httpd_custom_api.html

nice!